src/data/repositories/OptionsD2Repository.ts

```diff
@@ -6,12 +6,34 @@ import { OptionsRepository } from "../../domain/repositories/OptionsRepository";
 import { assertOrError } from "./utils/AssertOrError";
 import { getHazardTypeByCode } from "./consts/DiseaseOutbreakConstants";
 
+import {
+    Capability1,
+    Capability2,
+    HighCapacity,
+    HighGeographicalSpread,
+    HighPopulationAtRisk,
+    HighWeightedOption,
+    LowCapacity,
+    LowGeographicalSpread,
+    LowPopulationAtRisk,
+    LowWeightedOption,
+    MediumCapacity,
+    MediumGeographicalSpread,
+    MediumPopulationAtRisk,
+    MediumWeightedOption,
+    RiskAssessmentGrading,
+} from "../../domain/entities/risk-assessment/RiskAssessmentGrading";
+import { Future } from "../../domain/entities/generic/Future";
+
 const MAIN_SYNDROME_OPTION_SET_CODE = "AGENTS";
 const SUSPECTED_DISEASE_OPTION_SET_CODE = "RTSL_ZEB_OS_DISEASE";
 const NOTIFICATION_SOURCE_OPTION_SET_CODE = "RTSL_ZEB_OS_SOURCE";
 
 export class OptionsD2Repository implements OptionsRepository {
```

Comment:
We'll need to refactor the options repository as soon as possible and create an entity for each option (with its corresponding repo) and join the options in the corresponding hook in the presentation layer.

Reply:
will be taken care of in upcoming refactor PR